Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -201,3 +205,50 @@ extension NIOHTTP2Handler {
         }
     }
 }
+
+extension InlineStreamMultiplexer {
+    func setStreamChannels(_ streamChannels: any ContinuationSanitizer) {
+        self.commonStreamMultiplexer.setStreamChannels(streamChannels)
+    }
+}
+
+extension NIOHTTP2Handler {
+    /// A variant of `NIOHTTP2Handler.StreamMultiplexer` which creates a child channel for each HTTP/2 stream and yields it to the supplied async continuation.
+
+    /// In general in NIO applications it is helpful to consider each HTTP/2 stream as an
+    /// independent stream of HTTP/2 frames. This multiplexer achieves this by creating a
+    /// number of in-memory `HTTP2StreamChannel` objects, one for each stream. These operate
+    /// on ``HTTP2Frame/FramePayload`` objects as their base communication
+    /// atom, as opposed to the regular NIO `SelectableChannel` objects which use `ByteBuffer`
+    /// and `IOData`.
+    ///
+    /// The stream channel objects are initialized upon creation using the supplied `streamStateInitializer` which returns a type
+    /// `StreamChannelType` which must match the communication atom (final handler's InboundOut/OutboundIn types). This type may
```

Comment:
I think we should rename `StreamChannelType` to be `InitializationResult`. `StreamChannelType` suggests it has to be a `Channel` which isn't the case; it can be anything.

I also don't think this documentation is accurate anymore: it it doesn't have to match the final handlers types.

Reply:
I changed the generic type to `Output` to match the server bootstrap bind method.